app/src/main/java/com/example/plantsapp/presentation/ui/plantdetail/PlantDetailViewModel.kt

```diff
@@ -7,10 +7,10 @@ import com.example.plantsapp.domain.model.Plant
 
 class PlantDetailViewModel : ViewModel() {
 
-    val plant: LiveData<Plant> = MutableLiveData(detailPlant)
-}
-
-private const val SEVEN_DAYS: Int = 7
-private const val PLANT_PICTURE = "https://www.vippng.com/png/detail/41-414674_house-plant-png.png"
+    private val _plant: MutableLiveData<Plant> = MutableLiveData()
+    val plant: LiveData<Plant> get() = _plant
 
-private val detailPlant = Plant("Bob", "Succulent", PLANT_PICTURE, SEVEN_DAYS)
+    fun setPlant(plant: Plant) {
+        _plant.value = plant
+    }
```

Comment:
This is not good to: because even when you are off this screen, somehow this viewmodel is alive and store some Plant, but why it is needed at all?